tools/enso4igv/pom.xml

```diff
@@ -225,10 +225,28 @@
             <scope>test</scope>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.netbeans.api</groupId>
+            <artifactId>org-netbeans-modules-extexecution-base</artifactId>
+            <version>${netbeans.version}</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.api</groupId>
+            <artifactId>org-netbeans-modules-extexecution</artifactId>
+            <version>${netbeans.version}</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.api</groupId>
+            <artifactId>org-openide-io</artifactId>
+            <version>${netbeans.version}</version>
+            <type>jar</type>
+        </dependency>
     </dependencies>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <netbeans.version>RELEASE113</netbeans.version>
+        <netbeans.version>RELEASE140</netbeans.version>
```

Comment:
While working on this code base, I am trying to make sure the code works as:
-  [VSCode extension](https://github.com/enso-org/enso/blob/2f37be5ae9efa09bbae93121afa6ec910c59ec82/tools/enso4igv/README.md) 
- NetBeans module
- [IGV module](https://github.com/enso-org/enso/blob/2f37be5ae9efa09bbae93121afa6ec910c59ec82/tools/enso4igv/IGV.md)

The biggest challenge is support for [IGV](https://www.graalvm.org/latest/tools/igv/) as it is not being updated to run on [latest NetBeans version](https://netbeans.apache.org/front/main/download/nb20/) too frequently.

However for this PR I need [DialogDisplayer.notifyFuture](https://bits.netbeans.org/dev/javadoc/org-openide-dialogs/org/openide/DialogDisplayer.html#notifyFuture-T-) which is only available since _NetBeans 14_ - e.g. we have to upgrade.

Reply:
With [....](https://github.com/oracle/graal/pull/8290) the [Enso IGV Plugin vs. 1.31.129](https://github.com/enso-org/enso/actions/runs/7737409179?pr=8923) can be installed into **IGV**:

![Installation](https://github.com/enso-org/enso/assets/26887752/d592e8ea-fb69-4609-95df-82cfa79fb121)

Reply:
Once the `.nbm` file is installed into IGV, one can press **Shift-F6** inside of a `.enso` file and gets the same behavior as in the VSCode extension:

![Exec in IGV](https://github.com/enso-org/enso/assets/26887752/aeded3f1-427f-46bd-b362-baf8ecdeed32)